plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -327,7 +331,8 @@ const getFrontConfig = ( options = {} ) => {
 			// @see https://github.com/Automattic/jetpack/pull/20926
 			chunkFilename: `[name]-frontend${ fileSuffix }.js?ver=[contenthash]`,
 			filename: `[name]-frontend${ fileSuffix }.js`,
-			uniqueName: 'webpackWcBlocksJsonp',
+			uniqueName: 'webpackWcBlocksFrontendJsonp',
+			library: [ 'wc', '[name]' ],
```

Comment:
Webpack DEWP is actually non-functional in most of the bundles right now, because you must set a library export name if you want dependencies to be externalized. By setting this we enable DEWP on this bundle.

Reply:
@samueljseay can you help me understand this? What does it mean for DEWP to be non-functional in the bundles? As in it's not resolving imports `@woocommerce/blocks-checkout` to `window.wc.blocksCheckout`?

Reply:
@opr yes sorry this is sort of misleading, I shouldn't have said non-functional. 

DEWP will still resolve imports to a global variable if they already exist, but without this, it doesn't externalize anything in *this* bundle to the window. So if this bundle imports something already externalized in `main` and the main bundle is loaded it will get it from the window. But this bundle won't attach any dependencies to the window and I'm pretty sure that's not intentional because we declare blocks-checkout and blocks-components on the handles map: https://github.com/woocommerce/woocommerce/blob/faa170fb570e06b29d1867af32539542d1a44861/plugins/woocommerce-blocks/bin/webpack-helpers.js#L32-L33

Reply:
Thanks Sam, ah I see what you mean, so anything _we_ want to put onto the `winow.wc` wasn't going there from this package?